inmoose/consensus_clustering/consensus_clustering.py

```diff
@@ -349,6 +349,12 @@ def compute_clusters_consensus(self, prediction, k):
             ids = np.where(prediction == clust)[0]
             clust_size = len(ids)
             ids_ = np.array(list(combinations(np.sort(ids), 2))).T
+            if ids_.size == 0:
+                clusters_consensus[clust] = np.nan
+                LOGGER.error(
```

Comment:
should we log a warning or an error?
IMHO a warning would be more appropriate: as a user I do not control how my data will cluster. If there are single-sample clusters, I should take a closer look for sure (either the data or the number of clusters I requested)